src/components/group-add-post.js

```diff
@@ -168,13 +175,27 @@ class GroupAddPost extends React.Component {
                   required
                   error={error && errorExist(message, 'text')}
                 >
-                  <TextArea
-                    autoHeight
-                    rows={2}
-                    onChange={this.handleChange}
-                    name='text'
+                  <Editor
+	    	    apiKey="fb3pb0ana4mvi60jwhefs3g2u3501d9s915efud2rh6ax2ek"
+                    /*init={{height=autoHeight}}*/
+	    	    /*rows={2}*/
+                    onChange={this.handleEditorChange}
+                    textareaName="text"
                     value={this.state.text}
-                    placeholder='Write something here...'
+                    initialValue="<p>Write something here...</p>"
```

Comment:
Why was the `p` tags necessary?
Can't you simply write it like this - `initialValue="Write something here..."`?